package proto_ff;

import "yd_fieldoptions.proto";


message flashflashmonsterDesc
{
	optional int32 Num = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int64 ID = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message flashflash
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "刷新ID"];
	optional int32 isLevelDynamic = 2[(yd_fieldoptions.field_cname) = "是否动态等级"];
	optional int32 miniLevel = 3[(yd_fieldoptions.field_cname) = "动态初始等级"];
	optional int32 levelCheck = 4[(yd_fieldoptions.field_cname) = "动态等级校正"];
	optional int64 pointID = 5[(yd_fieldoptions.field_cname) = "坐标点ID"];
	optional int32 monster_Type = 6[(yd_fieldoptions.field_cname) = "怪物类型"];
	optional int32 monsterRefreshTime = 7[(yd_fieldoptions.field_cname) = "怪物刷新时间"];
	repeated flashflashmonsterDesc monster = 8[(yd_fieldoptions.field_cname) = "生物", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_flashflash
{
	repeated flashflash flashflash_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message flashitemFlashmonsterDesc
{
	optional int64 Id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 weight = 2[(yd_fieldoptions.field_cname) = "权重"];
}

message flashitemFlash
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "刷新ID"];
	optional string useMap = 2[(yd_fieldoptions.field_cname) = "可使用地图", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 live = 3[(yd_fieldoptions.field_cname) = "怪物存在时间"];
	optional int32 levelwave = 4[(yd_fieldoptions.field_cname) = "是否动态等级"];
	optional int32 flashNum = 5[(yd_fieldoptions.field_cname) = "刷新怪物数量"];
	repeated flashitemFlashmonsterDesc monster = 6[(yd_fieldoptions.field_cname) = "怪物", (yd_fieldoptions.field_arysize) = 10];
}


message Sheet_flashitemFlash
{
	repeated flashitemFlash flashitemFlash_List = 1[(yd_fieldoptions.field_arysize)=20];
}
